src/main/scala/io/treeverse/clients/ApiClient.scala

```diff
@@ -0,0 +1,25 @@
+package io.treeverse.clients
+
+import org.json4s._
+import org.json4s.native.JsonMethods._
+import scalaj.http.Http
+
+class ApiClient(apiUrl : String, accessKey : String, secretKey : String) {
+  def getRepositoryStorageNamespace(repoName: String): String = {
+    val resp = Http("%s/repositories/%s".format(apiUrl, repoName)).header("Accept", "application/json").auth(accessKey, secretKey).asString
```

Comment:
Can we construct the URL using `URL`?  Otherwise e.g. a trailing slash in `apiUrl` makes a difference, we depend more on characters allowed in repository names, and in general need to work harder.